packages/workspace/src/tasks-runner/default-reporter.ts

```diff
@@ -67,27 +68,38 @@ export class DefaultReporter {
           ]
         });
       }
-      return;
+    } else {
+      const bodyLines = [
+        output.colors.gray('Failed projects:'),
+        '',
+        ...failedProjectNames.map(
+          project => `${output.colors.gray('-')} ${project}`
+        )
+      ];
+      if (!affectedArgs.onlyFailed && !startedWithFailedProjects) {
+        bodyLines.push('');
+        bodyLines.push(
+          `${output.colors.gray(
+            'You can isolate the above projects by passing:'
+          )} ${output.bold('--only-failed')}`
+        );
+      }
+      output.error({
+        title: `Running target "${affectedArgs.target}" failed`,
+        bodyLines
+      });
     }
 
-    const bodyLines = [
-      output.colors.gray('Failed projects:'),
-      '',
-      ...failedProjectNames.map(
-        project => `${output.colors.gray('-')} ${project}`
-      )
-    ];
-    if (!affectedArgs.onlyFailed && !startedWithFailedProjects) {
-      bodyLines.push('');
-      bodyLines.push(
-        `${output.colors.gray(
-          'You can isolate the above projects by passing:'
-        )} ${output.bold('--only-failed')}`
-      );
+    if (cachedProjectNames.length > 0) {
+      const bodyLines = [
+        ...cachedProjectNames.map(
```

Comment:
Don't need this spread here right? Since it's already mapped from original array.

Reply:
You are right. I had extra lines there, but I removed them. The spread doesn't make much sense any more.